packages/share_plus/share_plus/example/lib/main.dart

```diff
@@ -77,6 +78,18 @@ class DemoAppState extends State<DemoApp> {
                 }),
               ),
               const SizedBox(height: 16),
+              TextField(
+                decoration: const InputDecoration(
+                  border: OutlineInputBorder(),
+                  labelText: 'Share uri',
+                  hintText: 'Enter the uri you want to share',
+                ),
+                maxLines: null,
+                onChanged: (String value) => setState(() {
+                  uri = value;
+                }),
```

Comment:
I would prefer a block body for `onChanged`.

```
onChanged: (String value) {
  setState(() => uri = value);
},
```

Reply:
The check take `share_plus_platform_interface: ^3.2.1` from pub.dev rather than your changes here.
Do we first need to deploy the PI, before the tests are passing, or could Melos fix it, @vbuberen?